### DISCUSSION TITLE
No release since December?

### DISCUSSION THREAD

**Discussion by mindplay-dk** [PrismJS/prism#2913] [General]
I picked Prism, since it lists `v` as a supported language on the home page.

So I set it all up, and it turns out, it does not highlight `v`, because that feature was never released - it was added some 4 months ago, but there's been no release since December last year.

I figured I'd pull it from Github, but then also noticed the minified version is out of date:

![image](https://user-images.githubusercontent.com/103348/119223740-5aed7080-bafb-11eb-9bbe-6e2b27704647.png)

It would be nice to be able to get the latest feature from a CDN.

Would you consider doing a weekly `latest` tagged release maybe?

**Comment by RunDevelopment** [comment]
> there's been no release since December last year

This is because we are currently doing the preparations for Prism v2. We internally decided that the next v1.x release will be the last Prism v1 version.

However, due to Prism's maintainers not having much time as of late, progress has been slower than expected.

> but then also noticed the minified version is out of date

It is not out of date. The commit to `prism-v.js` only affected spaces and comments. Therefore the minified file was not affected.

We also have a CI job that verifies that the minified files are up to date, so pulling from GitHub will always give you up-to-date minified files.

> Would you consider doing a weekly latest tagged release maybe?

What would be the advantage of this compared to pulling from GitHub?

**Reply by mindplay-dk** [reply]
> It is not out of date. The commit to `prism-v.js` only affected spaces and comments. Therefore the minified file was not affected.

Ah, gotcha 👍

> > Would you consider doing a weekly latest tagged release maybe?
> 
> What would be the advantage of this compared to pulling from GitHub?

CDNs like unpkg and skypack doin' their thang, so you can use it from jsfiddle or codesandbox 🙂

(I ended up using githack, but I always feel a little bad doing that, since it's one guy paying the bill.)

**Reply by RunDevelopment** [reply]
There are [npm tags](https://docs.npmjs.com/cli/v7/commands/npm-dist-tag) that we could use, but tags only point to a specific version. We would need something like a CI job that automatically makes new releases and then sets the tag.

Doable but I don't know if it's worth it for Prism. We don't have that many commits per week. The current situation (lots of commits but no release) is rather unusual and will go away after v2.0.0 anyway.

**Reply by RunDevelopment** [reply]
@mindplay-dk We released a new version yesterday. Just wanted to let you know.
